src/lib/components/dash_draggable.react.js

```diff
@@ -57,7 +57,12 @@ dash_draggable.propTypes = {
     /**
      * ...
      */
-    children: PropTypes.node
+    children: PropTypes.node,
+
+    /**
+     * ...
+     */
+    disabled: PropTypes.bool
```

Comment:
Thanks for adding this. Have you tested whether the component is working correctly? I would recommend adding an usage example with a button that let you enable or disable the component.

Reply:
It does work correctly, I will try to add the `callback` to `usage.py` at some point.